src/components/MoneyRequestHeader.tsx

```diff
@@ -52,7 +52,7 @@ function MoneyRequestHeader({report, parentReportAction, policy, shouldUseNarrow
     const [transaction] = useOnyx(`${ONYXKEYS.COLLECTION.TRANSACTION}${(parentReportAction as ReportAction & OriginalMessageIOU)?.originalMessage?.IOUTransactionID ?? 0}`);
     const [transactionViolations] = useOnyx(ONYXKEYS.COLLECTION.TRANSACTION_VIOLATIONS);
     const [session] = useOnyx(ONYXKEYS.SESSION);
-    const [shownHoldUseExplanation] = useOnyx(ONYXKEYS.NVP_HOLD_USE_EXPLAINED, {initWithStoredValues: false});
+    const [shownHoldUseExplanation] = useOnyx(ONYXKEYS.NVP_HOLD_USE_EXPLAINED, {initialValue: false});
```

Comment:
initWithStoredValues is "true by default", so we don't need to set it explicitly.

Btw, @DylanDylann do you know what issue this causes so I can test it?

Reply:
@bernhardoj The initial value should be true

Reply:
@bernhardoj What is difference between initWithStoredValues and initialValue

Reply:
If initWithStoredValues is false, it won't use the stored values, so it will return undefined, but it will still receive the update.

initialValue will be used if the data is not ready yet. So in this case, it will be false first and then true.